code/game/response_team.dm

```diff
@@ -112,7 +112,7 @@ var/send_emergency_team
 				return
 			var/client/C = src.client
 			var/mob/living/carbon/human/new_commando = C.create_response_team(L.loc)
-			del(L) // Do not qdel landmarks!
```

Comment:
Same line that @tigercat2000 commented on, but Github handles comments on commits badly. Did the comment here perhaps have a point to it?